src/pathways/pathway/dto/active-pathway.dto.ts

```
@@ -11,13 +12,23 @@ export class ActivePathwayDto {
+  @ApiPropertyOptional({
     description: "Optional Pathway UUID to check specific assignment status",
     example: "f365cc27-8e24-4065-829d-558f6a639f99",
-    required: false,
   })
   @Expose()
   @IsUUID()
   @IsOptional()
   pathwayId?: string;
+
+  @ApiPropertyOptional({
+    description: "Filter by pathway type. STANDARD (default) returns the single active career pathway. VOLUNTEER returns all active volunteer pathway assignments as an array. ALL returns both.",
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)